lib/cisco_node_utils/cmd_ref/tacacs_server_host.yaml

```diff
@@ -1,37 +1,66 @@
 # tacacs_server_host
 ---
-_exclude: [ios_xr]
+_template:
+  nexus:
+    get_command: "show run tacacs all"
+  ios_xr:
+    get_command: "show running-config tacacs-server"
 
 encryption:
-  set_value: "%s tacacs-server host %s key %s %s"
+  nexus:
+    set_value: "<state> tacacs-server host <ip> key <key>"
+  ios_xr:
+    context: ["tacacs-server host <ip> port <port>"]
+    get_value: '/key \d+\s+(\S+)/'
+    set_value: '<state> key <enc_type> <password>'
 
 encryption_password:
-  get_command: "show run tacacs all"
-  get_value: '/^tacacs-server host %s key \d+\s+(\S+)/'
   default_value: ""
+  nexus:
+    get_value: '/^tacacs-server host <ip> key \d+\s+(\S+)/'
+  ios_xr:
+    context: ["tacacs-server host <ip> port <port>"]
+    get_value: '/key \d+\s+(\S+)$/'
 
 encryption_type:
-  get_command: "show run tacacs all"
-  get_value: '/^tacacs-server host %s key (\d+)/'
+  nexus:
+    get_value: '/^tacacs-server host <ip> key (\d+)/'
+  ios_xr:
+    context: ["tacacs-server host <ip> port <port>"]
+    get_value: '/key (\d+)/'
 
 host:
-  set_value: "%s tacacs-server host %s"
+  nexus:
+    set_value: "<state> tacacs-server host <ip>"
+  ios_xr:
+    set_value: "<state> tacacs-server host <ip> port <port>"
+
+host_port_pairs:
+  _exclude: [nexus]
+  multiple: true
+  get_command: "show running-config tacacs-server"
+  get_value: '/^tacacs-server host <ip> port (\d+)/'
+  default_value: ''
 
 hosts:
   multiple: true
-  get_command: "show run tacacs all"
   get_value: '/^tacacs-server host (\S+)/'
+  default_value: ''
```

Comment:
Why the new default_value?  And why is the default an empty string when this will normally return an array?